app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        pytest.param(0, 0, [0, 0], id="get_from_zeroes"),
```

Comment:
Do not use `pytest.param` here, there is no need for that. Use just tuple in list, eg `[(0, 0, [0, 0]), (14, 14, [0, 0]), ...]`